src/pages/workspaces/workspace/WorkspaceTabs.test.ts

```diff
@@ -56,9 +88,20 @@ describe('WorkspaceTabs', () => {
 
   it('renders subset of tabs for Azure workspace with flag enabled, with no accessibility issues', async () => {
```

Comment:
I don't think there is a feature flag anymore… is this test point just duplicating the previous one?

Reply:
Looks like it.
Adding the only missing assertion to the previous test and removeding.